public/plugin/bbb/listing.php

```diff
@@ -182,12 +188,7 @@
             }
 
             $remoteId = Database::escape_string($_GET['remote_id']);
-            $meetingData = Database::select(
-                '*',
-                Database::get_main_table('plugin_bbb_meeting'),
-                ['where' => ['remote_id = ? AND access_url = ?' => [$remoteId, api_get_current_access_url_id()]]],
-                'first'
-            );
+            $meetingData = $meetingRepository->findOneByRemoteIdAndAccessUrl($remoteId, api_get_current_access_url_id());
 
```

Comment:
This do not return the values of the column user_id which is needed on line 199 and generated an error.
We should get an object user but I do not find it to be able to adapt line 199 so this still needs to be fixed.

Reply:
There is still a problem on line 196 because when returning from the video conference it does not find the $meetingData['id'] and gives a 500 error :
`PHP Fatal error:  Uncaught ErrorException: Warning: Undefined array key "id" in /var/www/.../public/plugin/bbb/listing.php:196`